.github/workflows/release-publish-suite-commit.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release-tests:
```

Comment:
how do we know this is running tests on the versions included in the release? what if the version in the release isn't the latest - will it still run the tests against those versions?